configs/helm/README.md

```diff
@@ -0,0 +1,66 @@
+# Telliot
```

Comment:
I want to avoid having a duplicate readme file. Is there any reason to keep this one?
Prefer to keep all docs in a single folder to make them easy to navigate and discover and also the docs are displayed in gitbook at https://docs.tellor.io/tellor/telliot/setup-and-usage